docs-client/package.json

```diff
@@ -9,56 +9,55 @@
     "start": "cross-env TS_NODE_PROJECT=\"tsconfig-webpack.json\" ts-node ./scripts/serve"
   },
   "dependencies": {
-    "@babel/polyfill": "7.0.0-beta.54",
-    "@material-ui/core": "1.4.0",
-    "@material-ui/icons": "1.1.0",
-    "jsonminify": "0.4.1",
-    "react": "16.4.1",
-    "react-dom": "16.4.1",
-    "react-helmet": "5.2.0",
-    "react-hot-loader": "4.3.3",
-    "react-router-dom": "4.3.1",
-    "react-syntax-highlighter": "8.0.1",
-    "typeface-roboto": "0.0.54",
-    "typeface-roboto-mono": "0.0.54"
+    "@babel/polyfill": "^7.0.0",
+    "@material-ui/core": "^3.3.2",
+    "@material-ui/icons": "^3.0.1",
+    "jsonminify": "^0.4.1",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-helmet": "^5.2.0",
+    "react-hot-loader": "^4.3.12",
+    "react-router-dom": "^4.3.1",
+    "react-syntax-highlighter": "^9.0.1",
+    "typeface-roboto": "^0.0.54",
+    "typeface-roboto-mono": "^0.0.54"
   },
   "devDependencies": {
-    "@babel/core": "7.0.0-beta.54",
-    "@babel/preset-env": "7.0.0-beta.54",
-    "@babel/preset-react": "7.0.0-beta.54",
-    "@babel/preset-stage-0": "7.0.0-beta.54",
```

Comment:
`preset-stage-*` has been dropped since Babel 7.0.0. I wasn't sure which plugins to apply, so I just added no plugins after removing this. Do you have suggestions for plugins? It seems to work without any plugins in `preset-stage-0` by the way.